Gemfile.lock

```diff
@@ -1,114 +0,0 @@
-GEM
```

Comment:
This file should not be deleted I think.

Reply:
Sure, added back. Anything else?